#! /bin/sh /usr/share/dpatch/dpatch-run
## 34_eximstatsmanpage.dpatch by Andreas Metzler <ametzler@downhill.at.eu.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add note about installing perl-modules on Debian to
## DP: generated manpage

diff -NurbBp exim.orig/src/eximstats.src exim/src/eximstats.src
--- exim.orig/src/eximstats.src	2009-10-19 14:26:34.000000000 +0200
+++ exim/src/eximstats.src	2009-11-15 12:16:19.000000000 +0100
@@ -500,6 +500,10 @@ To install these, download and unpack th
  make test
  make install
 
+On B<Debian GNU/Linux> you can use
+C<apt-get install libgd-perl libgd-text-perl libgd-graph-perl>
+instead.
+
 =item B<-chartdir>I <dir>
 
 Create the charts in the directory <dir>
